### PR TITLE
New dialects on mineires.json

### PR DESCRIPTION
**Pull request by b00tk1ll**
## Pre-Submission Checklist

- [x]You have only one commit (if not, squash them into one commit).
- [x]All new and existing tests pass the command `yarn test` and `yarn lint`.

## Description

added new dialects on mineires.json

* added 'Gastura'
* added 'Cacunda'
* added 'É mió'
* added 'Muié'
* added 'Embuchá'
* added 'Furdunço'

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -266,6 +266,15 @@
...
       ]
+    },
+    {
+      "login": "b00tk1ll",
+      "name": "Guilherme Silveira",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/8213622?v=4",
+      "profile": "http://twitter.com/b00tk1ll",
+      "contributions": [
+        "code"
+      ]
```
Comment (mvfsillva) [inline .all-contributorsrc]: Sorry, but remove this part, because I'll add you as a contributor.

I always do this when I release.

Comment (b00tk1ll) [reply .all-contributorsrc]: Ok, I made the change

File: dialects/mineires.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "examples": ["Arreda daqui"]
   },
   {
-    "dialect": "Cadin",
+    "dialect": "É mió",
```
Comment (mvfsillva) [inline dialects/mineires.json]: Why renaming this word?

Comment (b00tk1ll) [reply dialects/mineires.json]: I had typed it wrong, then I fixed

**Pull request merged**